projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.scss

```diff
@@ -25,6 +25,22 @@
   z-index: z-index(off-canvas);
 }
 
+.go-off-canvas.large {
+  width: 75vw;
+  background-color: $theme-light-app-bg;
+  color: $base-dark;
+  .go-off-canvas__header {
+    background-color: $base-light;
+    box-shadow: $global-box-shadow--regular;
+  }
+  .go-off-canvas__title {
+    color: $base-dark;
+  }
+  .go-off-canvas__content {
```

Comment:
Can we add some new lines above these class declarations please?

Reply:
Only cuz you said please 😄